XenModel/FriendlyNames.resx

```diff
@@ -1586,6 +1586,15 @@
   <data name="Message.help-host_sync_data_failed" xml:space="preserve">
     <value>Help on HA</value>
   </data>
+  <data name="Message.help-leaf_coalesce_completed" xml:space="preserve">

```

Comment:
Arent they needed to avoid debug assert that x doesn't exist in Friendly names?

Reply:
They would be needed if that code were hit, but it won't be hit since you have handled these cases already in Title and Description using entries from Messages.resx.

Reply:
wonderful understood